x-pack/plugins/cases/kibana.jsonc

```diff
@@ -25,7 +25,8 @@
       "management",
       "security",
       "notifications",
-      "ruleRegistry"
+      "ruleRegistry",
+      "files",
```

Comment:
hmm I wonder if the build system figures that out because the observability plugin doesn't have "actions" specified in either of their required or optional plugins yet they can still use connectors within cases right?

Reply:
If I remember correctly "actions" do not export any UI client. We fetch the connectors from our own API. Think about user profiles. We had to put the `security` everywhere so we use the `security` UI client. Basically, if we need a service through the `useKibana` hook it should be part of the `kibana.jsonc` file and passed in the `KibanaContextProvider`. Ideally, we should pass the case dependencies in our `CaseContextProvider` so consumers should not have to pass the dependencies themselves.

Reply:
Ah I see. I can go ahead and add it then 👍

Reply:
Security solution already has it as required to it's just observability that needs it.